openshift/backend.dc.json

```diff
@@ -192,16 +192,6 @@
                                     }
                                 ]
                             }
-                        },
-                        "mid": {
-                            "failurePolicy": "Abort",
-                            "execNewPod": {
-                                "command": [
-                                    "/usr/bin/container-entrypoint",
```

Comment:
does this not need to be executed anymore?

Reply:
No, replaced with something in the Jenkinsfile.